### PR TITLE
Update language_cz.h for new translations

### PR DESCRIPTION
**Pull request by clexpert**

### CODE REVIEW

File: Marlin/language_cz.h [hunk 1]
```diff
@@ -58,11 +61,13 @@
 #define MSG_PREHEAT_1                       _UxGT("Zahrat PLA")
 #define MSG_PREHEAT_1_N                     MSG_PREHEAT_1 _UxGT(" ")
 #define MSG_PREHEAT_1_ALL                   MSG_PREHEAT_1 _UxGT(" Vse")
+#define MSG_PREHEAT_1_END                   MSG_PREHEAT_1 _UxGT(" Tryska")
```
Comment (Kaibob2) [inline Marlin/language_cz.h]: MSG_PREHEAT_1_END should be translated to "End" or "finished".
I guess "Tryska" means "Nozzle"

File: Marlin/language_cz.h [hunk 2]
```diff
@@ -58,11 +61,13 @@
...
 #define MSG_PREHEAT_1_BEDONLY               MSG_PREHEAT_1 _UxGT(" Podloz")
 #define MSG_PREHEAT_1_SETTINGS              MSG_PREHEAT_1 _UxGT(" Nast")
 #define MSG_PREHEAT_2                       _UxGT("Zahrat ABS")
 #define MSG_PREHEAT_2_N                     MSG_PREHEAT_2 _UxGT(" ")
 #define MSG_PREHEAT_2_ALL                   MSG_PREHEAT_2 _UxGT(" Vse")
+#define MSG_PREHEAT_2_END                   MSG_PREHEAT_2 _UxGT(" Tryska")
```
Comment (Kaibob2) [inline Marlin/language_cz.h]: MSG_PREHEAT_2_END should be translated to "End" or "finished".
I guess "Tryska" means "Nozzle"

**Review by Kaibob2 — CHANGES_REQUESTED** [review]

File: Marlin/language_cz.h (hunk 1 above)
Comment (Kaibob2) [reply Marlin/language_cz.h]: Uhh, dammit, complete misunderstanding on my side. I derived it from language_en.h and interpreted MSG_PREHEAT_1_**END** as "finished" and not as "hotend".
Sorry for that.

File: Marlin/language_cz.h (hunk 2 above)
Comment (Kaibob2) [reply Marlin/language_cz.h]: Uhh, dammit, complete misunderstanding on my side. I derived it from language_en.h and interpreted MSG_PREHEAT_1_**END** as "finished" and not as "hotend".
Sorry for that.

Comment (clexpert) [reply Marlin/language_cz.h]: @Kaibob2 You are right. Thanks. I thought that "end" is abbreviation of "hotend" in this sense.

**Review by clexpert — COMMENTED** [review]
@Kaibob2 OK 😄 😄 😄 I have reverted the changes back to "hotend".

**Comment by Roxy-3D** [comment]
Thank You!

**Pull request merged**